corehq/privileges.py

```diff
@@ -99,4 +99,13 @@ def get_name_from_privilege(cls, privilege):
             REPORT_BUILDER: _('User Configurable Report Builder'),
             TEMPLATED_INTENTS: _('Templated Intents'),
             CUSTOM_INTENTS: _('Custom Intents'),
-        }.get(privilege, privilege)
+            DATA_CLEANUP: _('Data Management'),
+        }
+        if privilege in titles:
+            return titles[privilege]
+        else:
+            from corehq.util.soft_assert import soft_assert
+            _assert = soft_assert(notify_admins=True)
+            _assert(False, '{} does not have a title defined, users are '
+                           'seeing the slug itself.'.format(privilege))
```

Comment:
any sense how frequently this function is called? if we have issues here and start sending emails all the time in the webworker threads then HQ is going to slow down a lot for people